src/style/mixins.scss

```diff
@@ -186,3 +186,99 @@
     }
   }
 }
+
+@mixin iui-label-with-sublabel($label-font-size: $iui-font-size, $sublabel-font-size: $iui-font-size-small) {
```

Comment:
I do not think we should be putting these mixins here (especially the `iui-thumbnail-or-icon`) because currently it still feels very specific to header.

And when we want to create a generic solution, we might want to make it a utility component similar to `iui-surface`.

Reply:
I removed these 2 universal mixins and placed them in `header-buttons.scss`.  We can revisit making these parts more easily used in other components in a separate PR.